src/index.ts

```diff
@@ -1,13 +1,14 @@
-import {useDimensions} from './useDimensions'
-import {useAppState} from './useAppState'
-import {useBackHandler} from './useBackHandler'
-import {useCameraRoll} from './useCameraRoll'
-import {useClipboard} from './useClipboard'
-import {useAccessibilityInfo} from './useAccessibilityInfo'
-import {useKeyboard} from './useKeyboard'
-import {useInteractionManager} from './useInteractionManager'
-import {useDeviceOrientation} from './useDeviceOrientation'
-import {useLayout} from './useLayout'
+import useDimensions from './useDimensions'
+import useAppState from './useAppState'
+import useBackHandler from './useBackHandler'
+import useCameraRoll from './useCameraRoll'
+import useClipboard from './useClipboard'
+import useAccessibilityInfo from './useAccessibilityInfo'
+import useKeyboard from './useKeyboard'
+import useInteractionManager from './useInteractionManager'
+import useDeviceOrientation from './useDeviceOrientation'
+import useLayout from './useLayout'
+import useImageDimensions from './useImageDimensions'
```

Comment:
I think that this will break all other imports? 🤔

Reply:
Fixed